tests/jscs-test.js

```diff
@@ -1,14 +1,39 @@
 'use strict';
 
-var path = require('path');
 var walkSync = require('walk-sync');
+var execFile = require("child_process").execFile;
 
-var nodeTestFiles = walkSync(__dirname)
+var nodeTestFiles = walkSync('.')
   .filter(function(file) {
-    return /-test.js$/.test(file);
+    return /.js$/.test(file);
   })
-  .map(function(file) {
-    return path.join(__dirname, file);
+  .filter(function(file) {
+    return !/node_modules|tmp/.test(file);
+  })
+  .filter(function(file) {
+    return !/tests\/fixtures\//.test(file);
   });
 
-require('mocha-jscs')(['./index.js'].concat(nodeTestFiles));
+describe('jscs', function () {
+  nodeTestFiles.forEach(function(path) {
+
+    it("should pass for " + path, function (done) {
+      var error = new Error('');
+
+      execFile(
```

Comment:
FYI, if we care to have the addon's internal tests run on Windows, we'll have to modify the way we invoke JSCS. See https://github.com/tomchentw/mocha-jscs/issues/3. I'll submit a PR there after I confirm the fix would still work ok on OSX, but it's essentially doing this instead:

```javascript
var jscsPath = require.resolve('jscs/bin/jscs');

execFile(
  'node',
  [jscsPath, '--verbose', '--config', '.repo-jscsrc'],
  onProcessFinished
);
```

That said, our previous releases did not work on Windows either because of this, so don't feel compelled to fix it in this PR necessarily. I just thought I'd point it out but I can open a separate PR for `ember-suave` to address this (and any other Windows path related issues that might exist).